docs/platforms/java/common/configuration/options.mdx

```diff
@@ -162,6 +162,14 @@ _(New in version 6.0.0)_
 
 </ConfigKey>
 
+<ConfigKey name="enable-backpressure-handling">
+
+Set this boolean to `false` to disable automatic downsampling of transactions while the system is under load. The `tracesSampleRate` is halved for every failing health check up to 10 times, (roughly 0.001% of the original `tracesSampleRate`). Any positive health check will reset `tracesSampleRate` to its original value. Health checks run in the background every 10 seconds, checking for queue drops and rate limiting. Note: Before version 7.8.0 backpressure handling was disabled by default.
```

Comment:
(l) Hmm, does that make it easier to understand? I was initially confused about the note. But am not sure if that is much better tbh.

```suggestion
Set this boolean to `false` to disable automatic downsampling of transactions while the system is under load. The `tracesSampleRate` is halved for every failing health check up to 10 times, (roughly 0.001% of the original `tracesSampleRate`). Any positive health check will reset `tracesSampleRate` to its original value. Health checks run in the background every 10 seconds, checking for queue drops and rate limiting. Note: Starting with version 7.8.0 backpressure handling has changed from opt-in to opt-out.
```